translations/en-GB.json

```
@@ -558,22 +558,22 @@
   "INVALID_PREMIUM_CODE": "That premium code is not valid or has already been redeemed.",
   "SUCCESSFULLY_REDEEMED_PREMIUM_CODE": "Successfully redeemed a Starboard Premium code `{code}`. This server now has Starboard Premium for {duration}!",
   "PREMIUM_WILL_BE_ACTIVATED": "{type, select, code {This premium code will be activated} pledge {Your premium pledge will come into effect}} when this server's current Starboard Premium runs out.",
-  "NOT_A_PATRON": "You are not currently a patron and have no Starboard Premium to redeem. If you are a patron, make sure you've [connected your patreon account to your Discord account](https://support.patreon.com/hc/en-us/articles/212052266-Getting-Discord-access).",
+  "NOT_A_PATRON": "You are not currently a patron and have no Starboard Premium to redeem. If you are a patron, make sure you've [connected your Patreon account to your Discord account](https://support.patreon.com/hc/en-us/articles/212052266-Getting-Discord-access).",
   "ALREADY_REDEEMED_PREMIUM": "It seems you've already redeemed premium in one of your servers.",
   "PREMIUM_PENDING": "It seems your Patron payment is currently pending, please try again in a few minutes.",
   "PATRON_PAYMENT_DECLINED": "It seems your Patron payment has declined and you cannot redeem Starboard Premium until that is resolved.",
   "SUCCESSFULLY_REDEEMED_PREMIUM": "Successfully redeemed Starboard Premium for this server!",
-  "OLD_PREMIUM_FEATURES_RESTORED": "This server had Starboard Premium before, and had items (overrides, reward roles, emojis etc.) that exceeded the free limit and got taken away when the premium expired, so I've gone and restored them back.",
+  "OLD_PREMIUM_FEATURES_RESTORED": "This server had Starboard Premium before, and had items (overrides, reward roles, emojis, etc.) that exceeded the free limit and were removed when the premium expired, so I've gone and restored them back.",
```

```suggestion
  "OLD_PREMIUM_FEATURES_RESTORED": "This server had Starboard Premium before, and had items (overrides, reward roles, emojis, etc.) that exceeded the free limit and got taken away when the premium expired, so I've gone and restored them back.",
```
Sorta prefer the original, differentiating from actual removal of items e.g. filters or emojis, versus the action of withdrawing or seizing